test/cli-test.js

```diff
@@ -91,10 +91,11 @@ test('cli - hash - no password', function (t){
   var stdin = execCli(['hash'], function (err, stdout, stderr){
     t.ifError(err);
 
-    var actual = stderr.trim();
-    var expected = '[Error: Password must be a  non-empty string.]';
+    var expected = /Error: Password must be a non-empty string/;
 
-    t.is(actual, expected);
+    t.throws(function (){
```

Comment:
Why not test against the `stderr.message` property? As you have demonstrated here, `.throws()` is for testing whether or not a function throws. What we really want to test is whether or not we get the expected error message, right?